data/deprecated.json

```diff
@@ -1940,5 +1940,21 @@
   {
     "old": {"memorial:text": "*"},
     "replace": {"inscription": "$1"}
+  },
+  {
+    "old": {"zone:maxspeed": "DE:urban"},
+    "replace": {"maxspeed:type": "DE:urban"}
+  },
+  {
+    "old": {"zone:maxspeed": "DE:rural"},
+    "replace": {"maxspeed:type": "DE:urban"}
+  },
+  {
+    "old": {"zone:maxspeed": "30"},
+    "replace": {"zone:maxspeed": "DE:30"}
+  },
```

Comment:
```suggestion
```
This would apply to all places around the world with `zone:maxspeed=30`. It would be easier to mass edit it with JOSM.